custom_integrations/airzonecloud

```diff
@@ -0,0 +1 @@
+../core_brands/airzone
```

Comment:
Custom integration are not allowed to be symlinked and, also, do not support brands.

Reply:
Sorry!
Fixed in ae7b3418c1dd7a89004f5d989b834189fb1254ee